### PR TITLE
refactor links loader

### PR DESCRIPTION
**Pull request by MDunitz**
## Need
- to speed up loading of process links in order to speed up etl job

## approach
- dont read from db when loading bundles
- insert into `process_join_table` in the finalizer step of the ETL job

## Operator Instructions
Instructions to the operator: After merging, run python -m dcpquery.db --db=remote migrate to apply migrations in the corresponding environment.
Database migration is required and the integration tests will fail until this command is run.

This is a breaking change, it will either be necessary to drop the entire db and reload it or drop the `process_join_table` and run the `dcpquery_etl_finalizer` to recreate it with the unique constraint

### CODE REVIEW

File: dcpquery/etl/__init__.py [hunk 1]
```diff
@@ -110,7 +110,44 @@ def load_bundle(self, bundle, extractor=None, transformer=None):
...
+                         output_files_table.parent_process
+                  from input_files_table, output_files_table
+       where input_files_table.input_file_uuid = output_files_table.output_file_uuid) as temp_tables
+       ON CONFLICT DO NOTHING;
+        """
+    )
+    config.db_session.commit()
+
+
+def query_service_finalizer(extractor):
```
Comment (kislyuk) [inline dcpquery/etl/__init__.py]: Please rename this to a verb-based name, like `dcpquery_etl_finalize()`.

File: dcpquery/etl/__init__.py [hunk 2]
```diff
@@ -110,7 +110,44 @@ def load_bundle(self, bundle, extractor=None, transformer=None):
         config.db_session.commit()
 
 
-def create_view_tables(extractor):
+def update_process_join_table():
+    config.db_session.execute(
+        """
+        Insert into process_join_table (child_process_uuid, parent_process_uuid)
+select * from (
```
Comment (kislyuk) [inline dcpquery/etl/__init__.py]: Please fix the indentation here to be consistent with the other lines.

File: dcpquery/etl/__init__.py [hunk 3]
```diff
@@ -110,7 +110,44 @@ def load_bundle(self, bundle, extractor=None, transformer=None):
...
+                  input_files_table as (
+                                        select
+                                               process_uuid as child_process,
+                                               file_uuid    as input_file_uuid
+                                        from process_file_join_table
+                                        where process_file_connection_type = 'INPUT_ENTITY'
+                                      ),
+
+                  output_files_table as (
+                                          SELECT
```
Comment (kislyuk) [inline dcpquery/etl/__init__.py]: Please fix capitalization to be consistent within the SQL statement.

**Comment by kislyuk** [comment]
Tests are failing - please take a look.

File: dcpquery/etl/__init__.py (hunk 1 above)
Comment (MDunitz) [reply dcpquery/etl/__init__.py]: 👍

File: dcpquery/etl/__init__.py (hunk 2 above)
Comment (MDunitz) [reply dcpquery/etl/__init__.py]: 👍

File: dcpquery/etl/__init__.py (hunk 3 above)
Comment (MDunitz) [reply dcpquery/etl/__init__.py]: 👍

**Pull request merged**